src/hooks/useWithdrawLiquidityHandler.tsx

```diff
@@ -34,13 +35,24 @@ export const useWithdrawLiquidityHandler = () => {
   }
 
   async function approveTransaction() {
-    // Deposit pool contract address
-    let address = process.env.NEXT_PUBLIC_DEPOSIT_POOL_ADDRESS
-    if (provider && address) {
-      if (accountInfo && accountInfo?.address) {
-        setdepPool(new ethers.Contract(address, DepPool.abi, provider.getSigner(accountInfo?.address)))
+    // Position Manager contract address
+    let address =
+      process.env.NEXT_PUBLIC_ENVIRONMENT === 'local'
+        ? process.env.NEXT_PUBLIC_POSITION_MANAGER_ADDRESS || ''
+        : '0x3eFadc5E507bbdA54dDb4C290cc3058DA8163152'
```

Comment:
it's not good practice to declare strings without variables. Make sure global variables (if they're not an environment variables) are declared at the top of the file and defined with all caps.

Reply:
Added it as a separate variable